src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy;
+
+import java.util.Map;
+
+public interface OperationHandler {
+
+    void process(String fruit, int quantity, Map<String, Integer> storage);
```

Comment:
In all implementations of this interface, you should use Storage.fruits, which will be located in `core.basesyntax.db` package
Also, you already have FruitTransaction, use it 
```suggestion
    void process(FruitTransaction transaction);
```